app/helpers/decidim/problems/application_helper.rb

```diff
@@ -5,6 +5,7 @@ module Problems
     # Custom helpers, scoped to the problems engine.
     #
     module ApplicationHelper
+      include PaginateHelper
```

Comment:
can't we add it to the controllers that really need it as a helper instead of in the ApplicationHelper?